### ISSUE TITLE
Leverage scrollbars

### ISSUE THREAD

**Issue by cmdalbem** [zzarcon/react-scroll-shadow#7]
Congrats for the work and thanks for sharing with the community! :) This is a really interesting idea for a component, that I've seen myself reimplement in a couple of projects already.
However I noticed it doesn't handle well the case where scrollbars are visible, which would be a common case whenever the user has a mouse attached in a Mac, and probably even more common in Windows or other OS.

![image](http://g.recordit.co/je8PN0r2jQ.gif)

**Comment by dmackerman** [comment]
Yeah, not entirely sure why this library is called `scroll-shadow` when it doesn't actually use shadows or masking. 🤷
